src/components/sidebar/common/SectionResult.js

```diff
@@ -22,10 +22,19 @@ export default function SectionResult({ addSection, section, disabled }) {
   const classes = useStyles();
 
   const message = 'Class successfully added';
+  const handleUndoClick = () => {
+    dispatch(removeSection(section.id, section.color));
+    enqueueSnackbar('Class successfully removed', {
+      variant: 'success',
+    });
+  };
+  const action = () => <Button onClick={handleUndoClick}>Undo</Button>;
```

Comment:
Let's find a better name for this - this JSX isn't really the action. Clicking the button does dispatch an action, but I'd rather call this `undoButton`.